### PR TITLE
Update script.sh.template

### PR DESCRIPTION
**Pull request by MichalPetryka**

### CODE REVIEW

File: Runner/script.sh.template [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 cd runtime
 git log -1
+chmod 777 build.sh
 
 git remote add other https://github.com/{{SOURCE_REPOSITORY}}
 git fetch other {{SOURCE_BRANCH}}
 git log other/{{SOURCE_BRANCH}} -1
-git checkout -b pr
+git switch -c pr
+git reset --hard main
```
Comment (MihaZupan) [inline Runner/script.sh.template]: When would this change anything?

File: Runner/script.sh.template [hunk 2]
```diff
@@ -17,17 +19,17 @@ mkdir ../artifacts-pr
...
 
 cp -r artifacts/bin/coreclr/linux.x64.Release/* ../artifacts-main
 cp -r artifacts/bin/runtime/net8.0-linux-Release-x64/* ../artifacts-main
 cp -r artifacts/bin/coreclr/linux.x64.Checked/* ../clr-checked-main
 
-git checkout pr
-./build.sh clr.corelib+clr.nativecorelib+libs.pretest -c Release
-./build.sh clr -c Checked
+git switch pr
+./build.sh clr+libs.pretest -c Release
```
Comment (MihaZupan) [inline Runner/script.sh.template]:
```suggestion
./build.sh clr+libs.pretest+libs -c Release
```

Let's hope the build infra is smart enough to skip most work if there are no changes 🤷‍♂️

**Review by MihaZupan — APPROVED** [review]

File: Runner/script.sh.template (hunk 1 above)
Comment (MihaZupan) [reply Runner/script.sh.template]: The `reset` I mean

Comment (MichalPetryka) [reply Runner/script.sh.template]: It's just to be 100% sure the baseline is the same.

File: Runner/script.sh.template [hunk 3]
```diff
@@ -17,17 +19,17 @@ mkdir ../artifacts-pr
...
 
 cp -r artifacts/bin/coreclr/linux.x64.Release/* ../artifacts-main
 cp -r artifacts/bin/runtime/net8.0-linux-Release-x64/* ../artifacts-main
 cp -r artifacts/bin/coreclr/linux.x64.Checked/* ../clr-checked-main
 
-git checkout pr
-./build.sh clr.corelib+clr.nativecorelib+libs.pretest -c Release
-./build.sh clr -c Checked
+git switch pr
+./build.sh clr+libs.pretest+libs -c Release
```
Comment (MihaZupan) [inline Runner/script.sh.template]:
```suggestion
./build.sh clr+libs -c Release
```

Let's see if anything blows up

**Pull request merged**